R/capture-requests.R

```diff
@@ -202,7 +202,15 @@ dbConnectTrace <- quote({
 })
 
 dbSendQueryTrace <- quote({
-  if (dittodb_debug_level(2)) {
+  if (is.null(.dittodb_env$db_path)) {
+    rlang::abort(
+      message = c("Database capture failed",
+                  "*" = "The database connection object was created before calling 'start_db_capturing()'",
+                  "*" = "Please close the connection and ensure it's created after calling 'start_db_capturing()'."),
+      call = NULL
+    )
+  }
+  if (dittodb_debug_level(2)) {get_dbname(list(...), drv = drv)
```

Comment:
```suggestion
  if (dittodb_debug_level(2)) {
```

Was this a typo / accidental inclusion? I suspect this might be what is causing the CI failures in https://github.com/ropensci/dittodb/actions/runs/8169212871/job/22332821815?pr=186#step:10:264

Reply:
lol who put that there? 👀